### PR TITLE
Respect vendor conventions

### PR DESCRIPTION
**Pull request by TomRK1089**
Resolves issue #37  by using `conventions` values from Brunch configuration.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "typescript-brunch",
-  "version": "2.3.0",
+  "version": "2.3.1",
```
Comment (shvaikalesh) [inline package.json]: Let's not increment version in PR.

Comment (TomRK1089) [inline package.json]: Sure thing.

**Comment by shvaikalesh** [comment]
Thanks for the PR, @TomRK1089.
@paulmillr, please release PATCH.

**Comment by TomRK1089** [comment]
@paulmillr is there a timeline patch releases follow for plugins? Looking to convert an existing codebase to Typescript which is using Brunch which is what led to me discovering this issue. Should we wait for the NPM release or use the version of the plugin from `master` here for the time being?

**Pull request merged**